src/back/auth/src/service/service.hpp

```diff
@@ -28,7 +29,7 @@ class Service final : public components::LoggableComponentBase {
 
 	std::string GetLoginUrl(const std::string& callbackUrl) const;
 	std::string GetLoginCompleteUrl(
-		const Tokens& tokens,
+		const TokensAndUserData& tokensAndUserData,
```

Comment:
Лучше имя переменной покороче придумывать, если только это не какая то очень специфическая переменная имя которой должно сообщать какую то важную инфу о ней.)
```suggestion
		const TokensAndUserData& data,
```

Reply:
поправил.